File: lista_exercicos_02/lista_gilbert_em_c/04.c
#include <stdio.h>
#include <math.h>

int main(){
  float b, h, area, perim, diag;

  printf("Digite a base e a altura do retangulo:\n");
  scanf("%f", &b);
  scanf("%f", &h);

  area = b*h;
  perim=(b*2)+(h*2);
  diag=sqrt((b*b)+(h*h));

  printf("Area = %.2f - Perimetro = %.2f - Diagonal = %.2f", area, perim, diag);

  return 0;
}

File: lista_exercicos_02/04.py
a,b=map(float,input().split())
diag=(a**2)+(b**2)
diag=diag**(1/2)
area=a*b
perimetro=(a*2)+(b*2)
print(diag)
print("Área = {} - Perímetro = {} - Diagonal = {}".format(area, perimetro, diag))